spec/controllers/openid_connect/configuration_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe OpenidConnect::ConfigurationController do
+  describe '#index' do
+    let(:json_response) { JSON.parse(response.body).with_indifferent_access }
+
+    it 'renders information about the OpenID Connect integration' do
+      get :index
+
+      aggregate_failures do
+        expect(json_response[:issuer]).to eq(root_url)
+        expect(json_response[:authorization_endpoint]).to eq(openid_connect_authorize_url)
+        expect(json_response[:token_endpoint]).to eq(openid_connect_token_url)
+        expect(json_response[:userinfo_endpoint]).to eq(openid_connect_userinfo_url)
+        expect(json_response[:response_types_supported]).to eq(%w(code))
+        expect(json_response[:grant_types_supported]).to eq(%w(authorization_code))
+        expected_loa = [
```

Comment:
I can see both points, but I would agree with Peter. As long as we have a test that makes sure the constant is set to the correct values, then we can refer to the constant in tests since the constant is unit tested.